Dockerfile.tests

```diff
@@ -30,5 +30,11 @@ USER node
 # Expose port
 EXPOSE 9090
 
+# Hack to bust the cache. Refer to run_tests.sh file.
+ARG CACHEBUST=1
+
+#Running the tests
+RUN npm test
+
 # Start app
-CMD [ "npm", "test" ]
\ No newline at end of file
+CMD [ "npm", "start" ]
```

Comment:
Required?

Reply:
Hey, this would be required when I add stress testing scripts.

Reply:
Remove it for now add it back in the PR that requires this